### DISCUSSION TITLE
get jsonpath of current cursor position

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvimdev/lspsaga.nvim#1292] [Ideas]
<img width="919" alt="image" src="https://github.com/nvimdev/lspsaga.nvim/assets/95092244/124252a2-eade-4037-944b-9ed45ad553ab">

Now I can get the info from `Breadcrumbs`, so wondering if I can get the jsonpath by some function?

**Comment by glepnir** [comment]
an api function get elements `require('lspsaga.symbol.winbar').get_bar`
